cli/src/main.rs

```diff
@@ -0,0 +1,41 @@
+use std::str::FromStr;
+
+use clap::{command, Parser, Subcommand};
+
+#[derive(Parser, Debug)]
+#[command(author, version, about, long_about = None)]
+#[command(propagate_version = true)]
+struct CliArguments {
+    #[command(subcommand)]
+    command: Command,
+}
+
+#[derive(Subcommand, Debug)]
+enum Command {
+    TestIssueCredential {
+        #[arg(short, long)]
+        input_file_path: String,
```

Comment:
since these names ultimately become command-line arguments/flags, it'd be good to make them shorter I think. this could just be `input_file`